tall_brain/Assignment_7/calculator.py

```diff
@@ -0,0 +1,41 @@
+class calculator:
+    def add(self, x, y):
+        ans = x + y
+        return ans
```

Comment:
You can refactor this to be cleaner. Just do return x + y. Otherwise, if you enjoy how pedantic this code is by doing the operation on one line and returning it on the next, that's ok too. In this case I personally would do the one liner but I could see a case being made for doing it in two lines.